Gemfile

```
@@ -1,38 +1,36 @@
 source "https://rubygems.org"
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+git_source(:github){|repo| "https://github.com/#{repo}.git"}
 ruby "2.7.0"
-gem "rails", "~> 6.0.4", ">= 6.0.4.1"
+gem "bootsnap", ">= 1.4.2", require: false
+gem "jbuilder", "~> 2.7"
 gem "puma", "~> 4.1"
+gem "rails", "~> 6.0.4", ">= 6.0.4.1"
 gem "sass-rails", ">= 6"
-gem "webpacker", "~> 4.0"
 gem "turbolinks", "~> 5"
-gem "jbuilder", "~> 2.7"
-gem "bootsnap", ">= 1.4.2", require: false
+gem "webpacker", "~> 4.0"
 group :development, :test do
-  gem "sqlite3", "~> 1.4"
   gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
   gem "rubocop", "~> 0.74.0", require: false
   gem "rubocop-checkstyle_formatter", require: false
   gem "rubocop-rails", "~> 2.3.2", require: false
+  gem "sqlite3", "~> 1.4"
 end
 group :development do
-  gem "web-console", ">= 3.3.0"
   gem "listen", "~> 3.2"
   gem "spring"
   gem "spring-watcher-listen", "~> 2.0.0"
+  gem "web-console", ">= 3.3.0"
 end
 group :test do
   gem "capybara", ">= 2.15"
-  gem "selenium-webdriver"
-  gem "webdrivers"
-  gem "rails-controller-testing", "1.0.5"
-  gem "minitest",                 "5.11.3"
-  gem "minitest-reporters",       "1.3.8"
   gem "guard",                    "2.16.2"
   gem "guard-minitest",           "2.4.6"
+  gem "minitest",                 "5.11.3"
```

cách 1 space sau dấu phẩy
```suggestion
  gem "minitest", "5.11.3"
```